mobile/packages/staking/governance/manager.ts

```diff
@@ -81,7 +81,6 @@ class Manager implements GovernanceManager {
     const response = await api.getStakingKeyState(stakeKeyHash)
 
     if (isLeft(response)) {
```

Comment:
here you can inject the logger as part of the manager and log it, cuz the factory is called from the client and the logger maybe provided